src/views/v2/tickets/item/content.php

```diff
@@ -50,8 +50,16 @@
 	return;
 }
 
-$this->template( 'v2/tickets/item/content/title' );
+$context = $this->get_values();
 
-$this->template( 'v2/tickets/item/content/description' );
+$this->template(
+	'v2/tickets/item/content/title',
+	[
+		'ticket'  => $ticket,
+		'post_id' => $post_id,
+	]
+);
 
-$this->template( 'v2/tickets/item/extra' );
+$this->template( 'v2/tickets/item/content/description', $context );
+
+$this->template( 'v2/tickets/item/extra', $context );
```

Comment:
This shouldn't be necessary, if it is, we should fix it in a future sprint as part of a Common thing. This workaround shouldn't be needed/used.